src/__tests__/fake-timers.js

```diff
@@ -1,20 +1,26 @@
 import {waitFor, waitForElementToBeRemoved} from '..'
 import {render} from './helpers/test-utils'
 
-async function runWaitFor({time = 300} = {}, options) {
+async function runWaitFor({time = 300, advanceTimers = true} = {}, options) {
   const response = 'data'
   const doAsyncThing = () =>
     new Promise(r => setTimeout(() => r(response), time))
   let result
   doAsyncThing().then(r => (result = r))
-
+  if (advanceTimers) {
+    jest.advanceTimersByTime(time)
```

Comment:
Is this reachable from non-jest environments or where the `jest` global isn't available?

Reply:
I don't think so.
But I was thinking that this shouldn't matter since our tests are run with jest?

Reply:
Oh, I didn't notice this was just a test file.